src/main/scala/nl.knaw.dans.dccd.convert/DccdConvertExportApp.scala

```diff
@@ -15,10 +15,216 @@
+      dcxSpatialY = (parsedMetadata \\ "lng").text
+
+      dcIdentifierSid = (parsedMetadata \\ "sid").text + "\n"
+      dcIdentifierIdentifier = (parsedMetadata \\ "identifier").text
+
+      dcxRelationQualifier = " 'references' "
+      dcxRelationTitle = "Digital Collaboratory for Cultural Dendrochronology (DCCD)"
+      dcxRelationLink = "https://dendro.dans.knaw.nl/dccd/project/" + (parsedMetadata \\ "sid").text
+
+      dcxRelationQualifier_2 = " 'IsFormatOf' "
+      dcxRelationTitle_2 = (parsedMetadata \\ "sid").text
+
+      dcType = "DATASET"
+
+      language = (parsedMetadata \\ "language").text
+      isoDefault = new Locale(language)
+      languageISO3 = isoDefault.getISO3Language
+
+      ddmAccessRights = "NO_ACCESS"
+      now = Calendar.getInstance().getTime
+      ddmAvailable = Format.format(now)
+
+      ddmAudience = "D37000"
+
+      printer.printRecord(dirr, dcTitle, depositorId, dcxCreatorOrganization, ddmCreated, dctRightsHolder, dcDescription, dcSubject, dctTemporal, dcxSpatialX, dcxSpatialY, dcIdentifierSid, dcxRelationQualifier, dcxRelationTitle, dcxRelationLink, dcxRelationQualifier_2, dcxRelationTitle_2, dcType, languageISO3, ddmAccessRights, ddmAvailable, ddmAudience)
+      printer.printRecord("", "", "", "", "", "", "", dcElementType, "", "", "", dcIdentifierIdentifier, "", "", "", "", "", "", "", "", "", "")
+      printer.printRecord("", "", "", "", "", "", "", dcObjectType, "", "", "", "", "", "", "", "", "", "", "", "", "", "")
+      printer.printRecord("", "", "", "", "", "", "", dcTypeMetadata, "", "", "", "", "", "", "", "", "", "", "", "", "", "")
+      printer.printRecord("", "", "", "", "", "", "", dcCategory, "", "", "", "", "", "", "", "", "", "", "", "", "", "")
+      printer.printRecord("", "", "", "", "", "", "", dcTaxon, "", "", "", "", "", "", "", "", "", "", "", "", "", "")
+
+      csvPrinterToFile.printRecord(dirr, dcTitle, depositorId, dcxCreatorOrganization, ddmCreated, dctRightsHolder, dcDescription, dcSubject, dctTemporal, dcxSpatialX, dcxSpatialY, dcIdentifierSid, dcxRelationQualifier, dcxRelationTitle, dcxRelationLink, dcxRelationQualifier_2, dcxRelationTitle_2, dcType, languageISO3, ddmAccessRights, ddmAvailable, ddmAudience)
+      csvPrinterToFile.printRecord("", "", "", "", "", "", "", dcElementType, "", "", "", dcIdentifierIdentifier, "", "", "", "", "", "", "", "", "", "")
+      csvPrinterToFile.printRecord("", "", "", "", "", "", "", dcObjectType, "", "", "", "", "", "", "", "", "", "", "", "", "", "")
+      csvPrinterToFile.printRecord("", "", "", "", "", "", "", dcTypeMetadata, "", "", "", "", "", "", "", "", "", "", "", "", "", "")
+      csvPrinterToFile.printRecord("", "", "", "", "", "", "", dcCategory, "", "", "", "", "", "", "", "", "", "", "", "", "", "")
+      csvPrinterToFile.printRecord("", "", "", "", "", "", "", dcTaxon, "", "", "", "", "", "", "", "", "", "", "", "", "", "")
+
+
+    }
+
+    csvPrinterToFile.flush()
+    printer.flush()
```

Comment:
You could put the flush-es inside the loop, if something fails we at least have the stuff that is being flushed.